### DISCUSSION TITLE
Installer guide

### DISCUSSION THREAD

**Discussion by meganewday** [twbs/bootstrap#35652] [Q&A]
Is there any guide for installation?

**Comment by florianlacreuse** [comment]
Hum, do you mean with a package manager? https://getbootstrap.com/docs/5.1/getting-started/download/#package-managers

**Comment by mrmonteith** [comment]
I'm not using a package manager or anything. I just downloaded the Bootstrap 5.0.2 zip file.  But nothing about how to set it up, any best practice, etc.   My only guess is I can probably rename it to just Bootstrap so it's not version related and don't have to change paths if I use a newer version later.  I will probably create a version note in the folder or something.

**Reply by JamesVanWaza** [reply]
@mrmonteith make sure its the most updated version `5.3.2`

**Comment by adydetra** [comment]
@meganewday just visit this page, that's enough. https://getbootstrap.com/docs/5.3/getting-started/download/

`CDN` via jsDelivr

```html
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL" crossorigin="anonymous"></script>
```

`NPM`

```js
npm install bootstrap@5.3.2
```

`Yarn`

```js
yarn add bootstrap@5.3.2
```

`Composer`

```js
composer require twbs/bootstrap:5.3.2
```
